test/specs/behaviors/testDefinitions.ts

```diff
@@ -86,6 +87,81 @@ definitions.push({
+      testHelper.convertToBooleanIfApplicable(valueOfAttributeToBeAdded),
+    )
+
+    const property = {}
+    property[baseProperty] = valueOfAttributeToBeAdded
+    const expectedResultAttributeNotDefined = parameters.behavior(property).attributes[component][
+      attributeToBeAdded
+    ]
+    expect(testHelper.convertToBooleanIfApplicable(expectedResultAttributeNotDefined)).toEqual(
+      testHelper.convertToBooleanIfApplicable(valueOfAttributeToBeAdded),
+    )
+  },
+})
+
+// Example: Adds attribute 'aria-disabled=true' based on the property 'disabled'. This can be overriden by directly providing 'aria-disabled' property to the component.
+definitions.push({
+  regexp: /Adds attribute '([a-z A-Z -]+)=([a-z 0-9]+)' based on the property '([a-z]+)'\. This can be overriden by directly providing '([a-z -]+)' property to the component\./g,
+  testMethod: (parameters: TestMethod) => {
+    const [attributeToBeAdded, valueOfAttributeToBeAdded, baseProperty, overridingProperty] = [
+      ...parameters.props,
+    ]
+    const propertyWithOverride = {}
+    propertyWithOverride[overridingProperty] = valueOfAttributeToBeAdded
+    const expectedResultAttributeDefined = parameters.behavior(propertyWithOverride).attributes
+      .root[attributeToBeAdded]
+    expect(testHelper.convertToBooleanIfApplicable(expectedResultAttributeDefined)).toEqual(
+      testHelper.convertToBooleanIfApplicable(valueOfAttributeToBeAdded),
+    )
+    const property = {}
+    property[baseProperty] = true
+    const expectedResultAttributeNotDefined = parameters.behavior(property).attributes.root[
+      attributeToBeAdded
+    ]
+    expect(testHelper.convertToBooleanIfApplicable(expectedResultAttributeNotDefined)).toEqual(
+      testHelper.convertToBooleanIfApplicable(valueOfAttributeToBeAdded),
+    )
+  },
+})
+
+// Example: Adds attribute 'aria-disabled=true' to 'trigger' component's part based on the property 'disabled'.
+definitions.push({
+  regexp: /Adds attribute '([a-z A-Z -]+)=([a-z 0-9]+)' to '([a-z -]+)' component's part based on the property '([a-z -]+)'\.[^a-z A-Z]/g,
+  testMethod: (parameters: TestMethod) => {
+    const [attributeToBeAdded, valueOfAttributeToBeAdded, component, propertyDependsOn] = [
+      ...parameters.props,
+    ]
+    const propertyWithAriaSelected = {}
+    propertyWithAriaSelected[propertyDependsOn] = true
+    const expectedResultAttributeDefined = parameters.behavior(propertyWithAriaSelected).attributes[
+      component
+    ][attributeToBeAdded]
+    expect(testHelper.convertToBooleanIfApplicable(expectedResultAttributeDefined)).toEqual(
+      testHelper.convertToBooleanIfApplicable(valueOfAttributeToBeAdded),
+    )
+  },
+})
+
 // Example: Adds attribute 'aria-hidden=true', if there is no 'alt' property provided.
 definitions.push({
   regexp: /Adds attribute '([\w\-\w \s*]+)=([a-z]+)', if there is no '[a-z]+' property provided\.+/g,
```

Comment:
don't see any place where the match of the third 'group' (`[a-z]+` - yes, this is not a group in the expression, thus I am using quotes) is used. Is it really irrelevant what we would match there for tests' logic? According to the text of the statement, it is not